ui/src/testing/stubs/egress-events-admin-api-stub.ts

```diff
@@ -8,15 +8,11 @@ import {
   UpdateEgressEventRequest,
 } from 'generated/fetch';
 
-import { stubNotImplementedError } from 'testing/stubs/stub-utils';
-
 export class EgressEventsAdminApiStub extends EgressEventsAdminApi {
   private nextEventId = 1;
 
   constructor(public events: EgressEvent[] = []) {
-    super(undefined, undefined, (..._: any[]) => {
-      throw stubNotImplementedError;
-    });
+    super(undefined);
```

Comment:
This ultimately extends BaseApi which only takes one parameter: a configuration object.